build.sbt

```diff
@@ -2684,14 +2705,16 @@ lazy val `engine-runner` = project
   .dependsOn(yaml)
   .dependsOn(pkg)
   .dependsOn(cli)
+  .dependsOn(`profiling-utils`)
   .dependsOn(`library-manager`)
-  .dependsOn(`language-server`)
   .dependsOn(`edition-updater`)
   .dependsOn(`runtime-parser`)
   .dependsOn(`logging-service`)
   .dependsOn(`logging-service-logback` % Runtime)
+  .dependsOn(`engine-runner-common`)
   .dependsOn(`polyglot-api`)
   .dependsOn(`enso-test-java-helpers`)
+  .dependsOn(`language-server` % Runtime)
```

Comment:
Probably because of changing the dependency to `Runtime` the license check insist on having different hashcode (unexpected), but no other changes to licenses needed (that's expected).  CCing @radeusgd. Adjusted in 7dc2170

Reply:
I guess in ideal world the hashcode would not change if the actual dependencies do not change. 

The way the hashcode is computed is just a 'heuristic' that is meant to ensure we run the license review after adding/removing dependencies.

Such changes may also trip this heuristic - but as you did - you can just re-run the workflow and it updates the hashcode without issues.